ext/scripts/install_scripts/parse_single_line_package_list_with_comments.sh

```diff
@@ -7,7 +7,6 @@ function parse_single_line_package_list_with_comments(){
     #   optional: end with a group of 1 or more whitespace followed by a # and than a sequence of any character
     #       -> comment in the end
     line="$1"
-    regex=""
```

Comment:
Unused

Reply:
The question I have, is this file used at all? And, if can we replace it with the perl stuff.

Reply:
I did a quick check, we still use it in install_packages_batch.sh, which is only being used in the r-3.5 -datascience flavor.

Reply:
yes, this is actually something we should replace, the question is, if we want to do it in this PR

Reply:
Created https://github.com/exasol/script-languages-release/issues/451 to fix this